beers/models.py

```diff
@@ -38,18 +40,45 @@ class BeerStyle(models.Model):
     category = models.ForeignKey(BeerStyleCategory, on_delete='CASCADE')
     tags = models.ManyToManyField(BeerStyleTag, blank=True)
 
-    ibu_low = models.PositiveSmallIntegerField(default=0)
-    ibu_high = models.PositiveSmallIntegerField(default=0)
-    srm_low = models.PositiveSmallIntegerField(default=0)
-    srm_high = models.PositiveSmallIntegerField(default=0)
+    ibu_low = models.PositiveSmallIntegerField(
+        'Minimum bitterness (International Bitterness Units)', default=0,
+    )
+    ibu_high = models.PositiveSmallIntegerField(
+        'Maximum bitterness (International Bitterness Units)', default=0,
+    )
+    srm_low = models.PositiveSmallIntegerField(
+        'Minimum color (Standard Reference Method)',
+        default=0,
+    )
+    srm_high = models.PositiveSmallIntegerField(
+        'Maximum color (Standard Reference Method)', default=0,
+    )
 
-    og_low = models.DecimalField(max_digits=4, decimal_places=3, default=0)
-    og_high = models.DecimalField(max_digits=4, decimal_places=3, default=0)
-    fg_low = models.DecimalField(max_digits=4, decimal_places=3, default=0)
-    fg_high = models.DecimalField(max_digits=4, decimal_places=3, default=0)
+    og_low = models.DecimalField(
+        'Minimum original specific gravity',
+        max_digits=4, decimal_places=3, default=0,
+    )
+    og_high = models.DecimalField(
+        'Maximum original specific gravity',
+        max_digits=4, decimal_places=3, default=0,
+    )
+    fg_low = models.DecimalField(
+        'Minimum final specific gravity',
+        max_digits=4, decimal_places=3, default=0,
+    )
+    fg_high = models.DecimalField(
+        'Maximum final specific gravity',
+        max_digits=4, decimal_places=3, default=0,
+    )
 
-    abv_low = models.DecimalField(max_digits=3, decimal_places=1, default=0)
-    abv_high = models.DecimalField(max_digits=3, decimal_places=1, default=0)
+    abv_low = models.DecimalField(
+        'Maximum alcohol content (% by volume)',
+        max_digits=3, decimal_places=1, default=0,
```

Comment:
Doesn't match definition in beer

Reply:
I talked about that in the other PR. Monkeynaut is 7.25% in Untappd, but the BJCP styles only go to tenths of a percent.

Reply:
Oh yeah, fine by me then.